templates/shared/issuelist.tmpl

```diff
@@ -118,6 +118,32 @@
 							</span>
 						</span>
 					{{end}}
+					{{if and $.BlockingDependenciesMap (index $.BlockingDependenciesMap .ID)}}
```

Comment:
What do you think about moving this whole new block into a new subtemplate?

Reply:
its done